packages/components/pie-modal/tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
+      "target": "ES6",
+      "module": "ES2022",
+      "lib": ["es2020", "DOM", "DOM.Iterable"],
+      "declaration": true,
+      "declarationMap": true,
+      "sourceMap": true,
+      "inlineSources": true,
+      "outDir": "./compiled",
+      "baseUrl": ".",
+      "paths": {
+        "@/*": ["./src/*"]
+      },
+      "rootDir": ".",
+      "strict": true,
+      "noUnusedLocals": true,
+      "noUnusedParameters": true,
+      "noImplicitReturns": true,
+      "noFallthroughCasesInSwitch": true,
+      "noImplicitAny": true,
+      "noImplicitThis": true,
+      "moduleResolution": "node",
+      "allowSyntheticDefaultImports": true,
+      "experimentalDecorators": true,
+      "forceConsistentCasingInFileNames": true
+  },
+  "include": ["src/**/*.ts","./declaration.d.ts", "test/**/*.ts"],
+  "exclude": []
+}
```

Comment:
missing newline at the end of the file. Recommend using `.editorconfig` so editors add a newline automatically ([example](https://github.com/game-ci/unity-builder/blob/main/.editorconfig))